docs/sources/setup/install/helm/deployment-guides/gcp.md

```diff
@@ -0,0 +1,625 @@
+
+1. To start we will need create a `.htpasswd` file with the username and password. You can use the `htpasswd` command to create the file:
+
+   {{< admonition type="tip" >}}
+    If you don't have the `htpasswd` command installed, you can install it using `brew` or `apt-get` or `yum` depending on your OS.
+   {{< /admonition >}}
+
+    ```bash
+    htpasswd -c .htpasswd <username>
+    ```
+    This will create a file called `auth` with the username `loki`. You will be prompted to enter a password.
+
+ 2. Create a Kubernetes secret with the `.htpasswd` file:
+
+    ```bash
+    kubectl create secret generic loki-basic-auth --from-file=.htpasswd -n loki
+    ```
+
+    This will create a secret called `loki-basic-auth` in the `loki` namespace. We will reference this secret in the Loki Helm chart configuration.
+  
+3. Create a `canary-basic-auth` secret for the canary:
+
+    ```bash
+    kubectl create secret generic canary-basic-auth \
+      --from-literal=username=<USERNAME> \
+      --from-literal=password=<PASSWORD> \
+      -n loki
+    ```
+    We create a literal secret with the username and password for Loki canary to authenticate with the Loki gateway.
+    **Make sure to replace the placeholders with your desired username and password.** 
+
+### Loki Helm chart configuration
+
+Create a `values.yaml` file choosing the configuration options that best suit your requirements. Below there is an example of `values.yaml` files for the Loki Helm chart in [microservices](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#microservices-mode) mode.
+
+```yaml
+loki:
+   schemaConfig:
+     configs:
+       - from: "2024-04-01"
+         store: tsdb
+         object_store: gcs
+         schema: v13
+         index:
+           prefix: loki_index_
+           period: 24h
+   storage_config:
+     gcs:
+       bucket_name: <CHUNK_BUCKET_NAME> # Your actual gcs bucket name, for example, loki-gcp-chunks
+   ingester:
+       chunk_encoding: snappy
+   pattern_ingester:
+       enabled: true
+   limits_config:
+     allow_structured_metadata: true
+     volume_enabled: true
+     retention_period: 672h # 28 days retention
+   compactor:
+     retention_enabled: true 
+     delete_request_store: gcs
+   ruler:
+    enable_api: true
+    storage_config:
+      type: gcs
+      gcs_storage_config:
+        region: <REGION> # The GCS region, for example europe-west4
+        bucketnames: <RULER_BUCKET_NAME> # Your actual gcs bucket name, for example, loki-gcp-ruler
+      alertmanager_url: http://prom:9093 # The URL of the Alertmanager to send alerts (Prometheus, Mimir, etc.)
+
+   querier:
+      max_concurrent: 4
+
+   storage:
+      type: gcs
+      bucketNames:
+        chunks: <CHUNK_BUCKET_NAME> # Your actual gcs bucket name, for example, loki-gcp-chunks
+        ruler: <RULER_BUCKET_NAME> # Your actual gcs bucket name, for example, loki-gcp-ruler
+      
+serviceAccount:
+ create: false
```

Comment:
Shouldn't this be set to `true`?  Down at line 447 you talk about how this section defines the IAM role for the service account.

Reply:
That particular line says that a service account should not be _created_ by the config, and instead an existing one should be used. Do you think it's worth adding a note to explain this?

Reply:
Yeah, worth adding a note for that.